hadoop-ozone/dist/src/main/smoketest/s3/objecthead.robot

```diff
@@ -60,3 +60,13 @@ Head non existing key
     ${result} =         Execute AWSS3APICli and checkrc    head-object --bucket ${BUCKET} --key ${PREFIX}/non-existent   255
                         Should contain          ${result}    404
                         Should contain          ${result}    Not Found
+
+
+HeadObject Should Return StorageClass
+                        Execute                            echo "Randomtext" > /tmp/testfile
+    ${put_result} =     Execute AWSS3APICli and checkrc    put-object --bucket ${BUCKET} --key ${PREFIX}/object-exist --body /tmp/testfile    0
+
+    ${result} =         Execute AWSS3APICli and checkrc    head-object --bucket ${BUCKET} --key ${PREFIX}/object-exist    0
```

Comment:
I think this can be check in `Head existing object`. We don't need another test for this.

Reply:
@chungen0126 Thanks for the review.

I have remove the new test and integrate checking to `Head existing object` test, please help review again, thanks!